virtio-devices/src/block.rs

```diff
@@ -309,9 +309,13 @@ impl BlockEpollHandler {
                         read_avg = if read_avg == u64::MAX {
                             latency * LATENCY_SCALE
                         } else {
-                            read_avg
-                                + ((latency * LATENCY_SCALE) - read_avg)
-                                    / (read_ops_last + read_ops.0)
+                            // Cumulative average is guaranteed to be
+                            // positive if being calculated properly
```

Comment:
I don't think you need this comment - could add an assert...

Reply:
The comment was put to explain why the `unwrap()` was used here, e.g. programming error (not an error to be handled/reported).